docs/tutorials/io.rst

```diff
@@ -35,38 +32,37 @@ touched. So, you can tickle your device to make it laugh like this::
         else:
             display.show(Image.SAD)
 
-With one hand, hold your device by the GND pin. Then, with your other hand,
-touch (or tickle) the 0 (zero) pin. You should see the display change from
-grumpy to happy!
+Jedną ręką trzymaj urządzenie za pin GND. Następnie, drugą ręką dotknij (lub połaskocz) 
+pin 0 (zero). Powinieneś zobaczyć na wyświetlaczu zmianę z miny smutnej na szczęśliwą!
 
-This is a form of very basic input measurement. However, the fun really starts
-when you plug in circuits and other devices via the pins.
+To jest tylko bardzo podstawowa forma sprawdzenia działania wejścia. Prawdziwa zabawa
+zaczyna się, gdy do pinów podłączysz obwody oraz inne urządzenia.
 
-Bleeps and Bloops
-+++++++++++++++++
+Piski i buczenia
+++++++++++++++++
 
-The simplest thing we can attach to the device is a Piezo buzzer. We're going
-to use it for output.
+Najprostszym urządzeniem, które możemy podłączyć do naszego urządzenia 
```

Comment:
Urządzeniem - urządzenia. Może jednak warto wyjątkowo zostawić "rzeczą", choć to kalka językowa, żeby nie było powtórzenia.